### ISSUE TITLE
Corsair Gaming K70 LUX Keyboard: V204 uses new led handling also

### ISSUE THREAD

**Issue by frickler24** [mattanger/ckb-next#95]
in #48 the following info came from a Keyboard:
```
...
[I] claiming interfaces. name=Corsair Gaming K70 LUX Keyboard, serial=bla, firmware=0204; Got >> 3<< as ep_str
...

feb 28 11:47:16 yggdrasil ckb-daemon[8197]: [E] loadrgb_kb (led_keyboard.c:224): Bad input header
feb 28 11:47:16 yggdrasil ckb-daemon[8197]: [E] loadrgb_kb (led_keyboard.c:228): color = 0, i = 1, mode = 0
feb 28 11:47:16 yggdrasil ckb-daemon[8197]: Input(Antwort): 0e ff 01 3c ff ff ff ff
feb 28 11:47:16 yggdrasil ckb-daemon[8197]: Output (Frage): 0e ff 01 3c
feb 28 11:47:22 yggdrasil ckb-daemon[8197]: [E] os_usbrecv (via led_keyboard.c:233): Connection timed out
feb 28 11:47:22 yggdrasil ckb-daemon[8197]: [W] _start_dev (device.c:45): Unable to load hardware profile
feb 28 11:47:22 yggdrasil kernel: usb 3-6: usbfs: USBDEVFS_CONTROL failed cmd ckb-daemon rqt 161 rq 1 len 64 ret -110
feb 28 11:47:22 yggdrasil ckb-daemon[8197]: [I] Setup finished for /var/run/ckb1
```

What does it mean? The daemon tries to read the LED profiles, stored to the KB. The protocol for this is fix, but changed on a K95 RGB between firmware v204 and 205. Here we have V204 and get the eror.
So we must change the usb_led_fix to differentiate not only the version number, but the version specific to the hardware.

**Comment by frickler24** [comment]
is the issue for #96

**Comment by frickler24** [comment]
Is fixed with #195 at last.
